package/CHANGELOG

```diff
@@ -13,11 +13,12 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 kain88-de,jdetle, fiona-naughton, richardjgowers
+??/??/16 tyler.je.reddy, kain88-de,jdetle, fiona-naughton, richardjgowers
```

Comment:
Convention so far has been that new names are appended instead of prepended.
While you're at it, can you please insert a space after *kain88-de*? Thanks!